app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] @Team Great work adding the login page, however the design slightly differs from the design given, check the comparison below:

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/katarighe/blog-app/assets/61408860/e99fbec7-d31c-41b2-bd32-e6e01f9284bd) | ![image](https://github.com/rbd3/Blog-App2/assets/61408860/99b58dfa-d192-4184-9bb0-55f5ae942ee5)